streaming/base/format/base/writer.py

```diff
@@ -55,26 +47,21 @@ class Writer(ABC):
             while writing). Defaults to ``0``.
         extra_bytes_per_sample (int): Extra bytes per serialized sample (for computing shard size
             while writing). Defaults to ``0``.
+        **kwargs (Any): Additional settings for the Writer.
```

Comment:
I think kwargs are `Dict[str, Any]`?

Reply:
User can pass the args directly as `progress_bar=True, max_workers=4`, etc. Hence, kept as Any.